### PR TITLE
Fix for object being passed instead of array

### PR DESCRIPTION
**Pull request by jonnywilliamson**
https://github.com/irazasyed/telegram-bot-sdk/commit/ae214d081928afd4dc9916493768158c28075790#commitcomment-17194269

### CODE REVIEW

File: src/Laravel/TelegramServiceProvider.php [hunk 1]
```diff
@@ -74,7 +74,7 @@ protected function registerManager(Application $app)
         $app->singleton('telegram', function ($app) {
             $config = $app['config']['telegram'];
 
-            return (new BotsManager($config))->setContainer($app);
+            return (new BotsManager((array)$config))->setContainer($app);
```
Comment (jonnywilliamson) [inline src/Laravel/TelegramServiceProvider.php]: Or should the cast to array happen on line 75?

Comment (irazasyed) [inline src/Laravel/TelegramServiceProvider.php]: Yes

**Comment by jonnywilliamson** [comment]
Cast to array placed when the `$config` if being created.

**Comment by jonnywilliamson** [comment]
Thank you!

**Pull request merged**